ItemPool.py

```diff
@@ -963,6 +963,13 @@ def get_pool_core(world: World) -> tuple[list[str], dict[str, Item]]:
         # Update pattern matcher since all normal junk is removed.
         item_groups['Junk'] = remove_junk_ludicrous_items
         world.distribution.distribution.search_groups['Junk'] = remove_junk_ludicrous_items
+        #Precomplete Dungeons with Ludicrious Item Pool
+        if world.settings.empty_dungeons_mode != 'none':
+            precomplete_locations: list[str] = [location in world.empty_dungeons]
```

Comment:
This is a list containing a single `bool`, not a list of strings. We should probably include type checks in CI.